sci-libs/openlibm/openlibm-0.7.2.ebuild

```diff
@@ -1,23 +1,34 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit eutils toolchain-funcs fortran-2
+inherit toolchain-funcs
 
 DESCRIPTION="High quality system independent, open source libm"
 HOMEPAGE="https://github.com/JuliaLang/openlibm"
-SRC_URI="https://github.com/JuliaLang/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/JuliaMath/openlibm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="public-domain MIT ISC BSD-2 LGPL-2.1+"
```

Comment:
When there's a lot of licences, I like to comment where they come from & why. Not sure if applicable here.

Reply:
I dunno why they are there, so I didn't remove them :confused: .

Reply:
Then go check what the licence should be! Maybe more got added, or fewer, or ...

Reply:
https://github.com/JuliaMath/openlibm/blob/master/LICENSE.md
This seems to be the reason.
 I don't think it needs to be commented if its present in the License file, which is quite straightforward to fine.